dojo/templates/base.html

```diff
@@ -440,28 +440,7 @@
                                                     <a rel="noopener noreferrer" href="{% url 'metrics_panel' %}">
                                                         {% trans "Metrics Panel" %}
                                                     </a>
-                                                </li> 
-                                                {% if "Metrics_Panel_Admin"|has_role_permission %}
-                                                    <li>
-                                                        <a rel="noopener noreferrer" href="{% url 'metrics_panel_admin' %}">
-                                                            {% trans "Metrics Panel Administrative" %}
-                                                        </a>
-                                                    </li>
-                                                {% endif %}  
-                                                {% if "Metrics_DevSecOps"|has_global_permission %}
-                                                    <li>
-                                                        <a rel="noopener noreferrer" href="{% url 'metrics_devsecops' %}">
-                                                            {% trans "Metrics DevSecOps" %}
-                                                        </a>
-                                                    </li>
-                                                {% endif %}
-                                                {% if "Metrics_Scan_Cycle"|has_role_permission %}
-                                                    <li>
-                                                        <a rel="noopener noreferrer" href="{% url 'metrics_scan_cycle' %}">
-                                                            {% trans "Metrics Scan Cycle" %}
-                                                        </a>
-                                                    </li>
-                                                {% endif %}                                                 
+                                                </li>
```

Comment:
please, the following options must be present:

1. Metrics panel
2. Administrative metrics dashboard
3. DevSecOps Metrics